src/views/SubmissionView/hooks/useSubmissionValidationStatus.test.js

```diff
@@ -52,7 +56,7 @@ describe('useSubmissionValidationStatus', () => {
     });
     const { validateBeforeConfirmation } = useSubmissionValidationStatus([''], []);
     validateBeforeConfirmation.useCallback.cb();
-    expect(state.setState.submissionTriggered).toHaveBeenCalledWith(true);
+    expect(setValue).toHaveBeenCalledWith(true);
```

Comment:
It'd be great to add a comment here similar to the ones added in `FileRenderer/hooks.test.js`.